falcon/test_support.py

```diff
@@ -0,0 +1,21 @@
+import string
+import random
+from tasks.models import Priority, Status
+
+
+class Random:
```

Comment:
when you are creating a class please try to do it as much as you can unique because the generic ones probably already exist and you/others can be confused between them or in the worse case to overwrite them.
for example:
https://docs.python.org/3/library/random.html#random.Random